src/ui/components/stepCard/StepCard.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { ReactElement } from 'react'
+import styled, { DefaultTheme, useTheme } from 'styled-components/native'
+
+import { getSpacing, Typo } from 'ui/theme'
+
+interface StepCardProps {
+  title: string
+  icon: ReactElement
+  subtitle?: string
+  disabled?: boolean
+}
+
+export const StepCard: React.FC<StepCardProps> = ({ title, subtitle, icon, disabled = false }) => {
+  const hasSubtitle = !!subtitle
+  const theme = useTheme()
+  const iconElement = getIconWithColors(icon, disabled, theme)
+
+  return (
+    <Container disabled={disabled} hasSubtitle={hasSubtitle} testID="stepcard-container">
+      <Icon testID="stepcard-icon">{iconElement}</Icon>
+      <TextContainter>
+        <Title disabled={disabled}>{title}</Title>
+        {!disabled && !!hasSubtitle ? <Subtitle>{subtitle}</Subtitle> : null}
+      </TextContainter>
+    </Container>
+  )
+}
+
+const Container = styled.View<{ disabled?: boolean; hasSubtitle?: boolean }>(
+  ({ theme, disabled, hasSubtitle }) => ({
+    flexDirection: 'row',
+    borderWidth: 1,
+    borderRadius: theme.borderRadius.radius,
+    paddingTop: hasSubtitle ? getSpacing(5) : getSpacing(6),
+    paddingBottom: hasSubtitle ? getSpacing(5) : getSpacing(6),
+    paddingLeft: getSpacing(3),
+    paddingRight: getSpacing(3),
+    borderColor: disabled ? theme.colors.greyMedium : theme.colors.black,
+    width: disabled ? '98%' : '100%',
+  })
+)
+
+const Icon = styled.View({
```

Comment:
ce n'est pas vraiment l'icône que tu stylises mais son wrapper :

```suggestion
const IconWrapper = styled.View({
```
ou
```suggestion
const IconContainer = styled.View({
```